src/Arcus.BackgroundJobs.DataBricks/JobRun.cs

```diff
@@ -0,0 +1,33 @@
+﻿using GuardNet;
+using Microsoft.Azure.Databricks.Client;
+
+namespace Arcus.BackgroundJobs.Databricks
+{
+    /// <summary>
+    /// Represents a Databricks <see cref="Run"/> in a Databricks job.
+    /// </summary>
+    public class JobRun
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="JobRun"/> class.
+        /// </summary>
+        public JobRun(string name, Run run)
```

Comment:
is this the job name or run name?

Reply:
Job name, the run name is in the `Run` model I guess. I should document this.

Reply:
I'll rename.